test/contract.js

```diff
@@ -78,15 +78,15 @@ describe('Contract', () => {
 
     it('returns the args object with the sent params', async () => {
       const res = {
-        "txid": "60c14ddc003a84a0947db1ac9cbfb01f1d3253b7948dc95343d7dd5a9d5900e9",
-        "sender": "qUNeKdqh269AwJWxDFnxUAqKSJ9deSUsgF",
-        "hash160": "76a177b79b8ef37437dce27a38fa2653eb6d8241",
-        "args": {
-          "contractAddress": "f6177bc9812eeb531907621af6641a41133dea9e",
-          "amount": 0,
-          "gasLimit": 250000,
-          "gasPrice": 4e-7
-        }
+        txid: '60c14ddc003a84a0947db1ac9cbfb01f1d3253b7948dc95343d7dd5a9d5900e9',
+        sender: 'qUNeKdqh269AwJWxDFnxUAqKSJ9deSUsgF',
+        hash160: '76a177b79b8ef37437dce27a38fa2653eb6d8241',
+        args: {
+          contractAddress: 'f6177bc9812eeb531907621af6641a41133dea9e',
+          amount: 0,
+          gasLimit: 250000,
+          gasPrice: 4e-7,
```

Comment:
nit: one more test case to cover the `fix(8)`?

Reply:
actually that is just the return value of the tx. in JS, it will return it as an exponential anyways even if we are passing in the string rep of 4e-7 `'0.0000004'`. so either way the return is always the same, only the input to qtum rpc call is different.